app/templates/pages/already-a-member.html

```diff
@@ -0,0 +1,28 @@
+{% extends "components/base.html" %}
+
+{% block pageTitle %}
+Already a Member
+{% endblock %}
+
+{% block beforeContent %}
+{{ super() }}
+{{ govukBackLink ({
+'text': 'Back',
+'href': '/'
+}) }}
+{% endblock %}
+
+{% block content %}
+
+{{ govukPanel({
+'titleText': 'It appears you are already a member of the MoJ GitHub Organisation. Please sign in to GitHub and check
```

Comment:
Shouldn't this state explicitly which org? `moj-analytical-services` or `ministryofjustice` or say 

```suggestion
'titleText': 'It appears you are already a member of the MoJ GitHub Organisation you attempted to join. Please sign in to GitHub and check
```